### PR TITLE
Site dashboard

### PR DESCRIPTION
**Pull request by tienhoah**

### CODE REVIEW

**Comment by tienhoah** [comment]
closes #28

File: src/components/Button.js [hunk 1]
```diff
@@ -1,9 +1,22 @@
 import styled from 'styled-components/macro';
 
 export const ButtonStyle = styled('button')`
+  transform: translate(0%);
+  transition: ${props => props.theme.transition};
   color: ${props => props.theme.fg};
-  border: 2px solid ${props => props.theme.fg};
+  border: ${props => (props.buttonBorder ? `1px solid ${props => props.theme.fg}` : 0)};
```
Comment (brianbancroft) [inline src/components/Button.js]: Another way of doing this: 

```js
import styled, { css } from 'styled-components/macro'

export const ButtonStyle = styled('button')`
  (...)
  ${props => props.buttonBorder && css`
    border: 1px solid ${props => props.theme.fg};
   }
  (...)
```

In this case, both are perfectly fine ways of doing it **and this doesn't require to be changed if you don't want to**. But when you have `2` or more rules that require the same if-statement, that's when using the `css` templating really shines!

File: src/components/Button.js [hunk 2]
```diff
@@ -1,9 +1,22 @@
...
+  position: ${props => props.theme.position};
+  padding: ${props => props.theme.padding};
+  width: ${props => props.theme.width};
+  box-shadow: ${props => props.theme.shadow};
+  cursor: pointer;
+  &:hover {
+    box-shadow: ${props =>
+      props.setHover ? '0 1px 3px rgba(0,0,0,0.12), 0 1px 2px rgba(0,0,0,0.24)' : null};
+    transform: ${props => props.theme.transform};
+    transition: ${props => props.theme.transition};
```
Comment (brianbancroft) [inline src/components/Button.js]: I would advise caution when abstracting "transition" and "transform" into the themeProvider. I've found that the thing that I'm transforming, or transitioning sometimes requires 0.05s difference from the other thing just to keep everything appearing to be in sync. This is something I've avoided doing in my atlasAI Project.

File: src/components/Card.js [hunk 3]
```diff
@@ -9,12 +9,15 @@ import BarChart from './BarChart';
 
 const cardStyle = makeStyles(theme => ({
   root: {
+    padding: '0 0 16px 0'
```
Comment (brianbancroft) [inline src/components/Card.js]: have you tried using just `paddingBottom: '16px',` here?

File: src/components/Card.js [hunk 4]
```diff
@@ -9,12 +9,15 @@ import BarChart from './BarChart';
...
+  },
+  cardProperty: {
     padding: theme.spacing(1, 1)
   }
 }));
 
 const Card = ({ content: { title, body, type } }) => {
-  const { root: paperClasses } = cardStyle();
+  const classes = cardStyle();
```
Comment (brianbancroft) [inline src/components/Card.js]: This is fine. Now that I'm seeing that you're using destructuring a lot more, I'll lay off, unless it's a situation that _really_ merits using it. 🌮

**Pull request merged**